app/uk/gov/hmrc/emailverification/models/VerifyEmailRequest.scala

```diff
@@ -27,15 +27,27 @@ case class VerifyEmailRequest(
     email:                     Option[Email],
     lang:                      Option[Language],
     backUrl:                   Option[String],
-    pageTitle:                 Option[String]
+    pageTitle:                 Option[String],
+    labels:                    Option[Labels]
 )
 
 case class Email(address: String, enterUrl: String)
 
+case class Label(pageTitle: Option[String], userFacingServiceName: Option[String])
+case class Labels(en: Label, cy: Label)
+
 object Email {
   implicit val format: Format[Email] = Json.format[Email]
 }
 
 object VerifyEmailRequest {
   implicit val reads: Reads[VerifyEmailRequest] = Json.reads[VerifyEmailRequest]
 }
+
+object Label {
+  implicit val reads: Reads[Label] = Json.reads[Label]
+}
+
+object Labels {
+  implicit val reads: Reads[Labels] = Json.reads[Labels]
+}
```

Comment:
The type parameters of `Json.format` and `Json.reads` can be inferred by the compiler here and so can be dropped.